programs/server/Server.cpp

```diff
@@ -1427,6 +1430,9 @@ try
 
     }
 
+    std::chrono::milliseconds startup_duration = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now() - startup_start_time);
+    ProfileEvents::increment(ProfileEvents::ServerStartupTime, startup_duration.count());
```

Comment:
It is not the right place, at this moment startup is not finished, AFAIK `servers_to_start_before_tables` are keeper servers, not the clickhouse server, need to put this lower - near line 1820.

Reply:
addressed it, thanks